styles/style.css

```diff
@@ -35,4 +35,12 @@ body button {
 #game-board {
     float: left;
     padding: 20px 0px;
+   
 }
+
+#canvas {
+  background-image: url('/lab-canvas-race-car/images/road.png');
```

Comment:
Another way to do that is creating the image inside of Game's constructor.

```
const backgroundImage = new Image();
backgroundImage.src = "../images/road.png"
this.imgRoad = backgroundImage;
```

And then drawing it inside of the update function

`this.ctx.drawImage(this.imgRoad, 0, 0, canvas.width, canvas.height)`

So the road image is only going to show up when you click the start button!